docs/yaml_docs/pipeline_clustering_yml.md

```diff
@@ -0,0 +1,273 @@
+         - <span class="parameter">mindist </span> `Float`, Default: 0.5<br>
+           Can specify an array: 0.25,0.5,0.8
+      - <span class="parameter">rna:</span>
+         - <span class="parameter">mindist </span> `Float`, Default: 0.5<br>
+            Can specify an array: 0.25,0.5,0.8
+
+## Parameters for clustering 
+
+  - <span class="parameter">clusterspecs:</span>
+      - <span class="parameter">rna:</span>
+          - <span class="parameter">resolutions </span> `Float`, Default: 0.2, 0.6, 1<br>
+           Can specify an array: 0.2,0.6,1
+          - <span class="parameter">algorithm</span> `String`, Default: leiden<br>
+            Options include louvain or leiden. 
+      - <span class="parameter">prot:</span>
+          - <span class="parameter">resolutions </span> `Float`, Default: 0.2, 0.6, 1<br>
+           Can specify an array: 0.2,0.6,1
+          - <span class="parameter">algorithm</span> `String`, Default: leiden<br>
+            Options include louvain or leiden.
+
+      - <span class="parameter">atac:</span>
+          - <span class="parameter">resolutions </span> `Float`, Default: 0.2, 0.6, 1<br>
+           Can specify an array to compute in parallel: 0.2,0.6,1
+          - <span class="parameter">algorithm</span> `String`, Default: leiden<br>
+            Options include louvain or leiden. 
+      - <span class="parameter">multimmodal:</span>
+          - <span class="parameter">resolutions </span> `Float`, Default: 0.5, 0.7<br>
+           Can specify an array to compute in parallel: 0.2,0.6,1 
+          - <span class="parameter">algorithm</span> `String`, Default: leiden<br>
+            Options include louvain or leiden.
+
+      - <span class="parameter">spatial:</span>
+          - <span class="parameter">resolutions </span> `Float`, Default: 0.2, 0.6, 1<br>
+           Can specify an array to compute in parallel: 0.2,0.6,1 
+          - <span class="parameter">algorithm</span> `String`, Default: leiden<br>
+            Options include louvain or leiden. 
+
+## Parameters for finding marker genes 
+If pseudo_suerat is set to false then we run [scanpy](https://scanpy.readthedocs.io/en/stable/generated/scanpy.tl.rank_genes_groups.html). 
+When pseudo_seurat is set to true then a python implementation of Suerat runs (Seurat::FindMarkers written by CRG)
+
+  - <span class="parameter">markerspecs:</span> <br>
+     - <span class="parameter">rna:</span><br>
+       - <span class="parameter">run </span> `Boolean`, Default: True<br>
+       - <span class="parameter">layer </span> `String`, Default: logged_counts<br>
+       - <span class="parameter">method </span> `String`, Default: t-test_overestim_var<br>
+       Options include: ‘logreg’, ‘t-test’, ‘wilcoxon’, ‘t-test_overestim_var’
+       - <span class="parameter">mincels </span> `Integer`, Default: t-10<br>
+       If the number of clusters contains less than the number of cells maker analysis is not necessary.
```

Comment:
i don't understand this, please check the documentation of scanpy.rank_genes to explain this parameter